core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.ts

```diff
@@ -496,15 +496,22 @@ export class StoryNodeEditorComponent implements OnInit, OnDestroy {
   updateLocalEditableOutline($event: string): void {
     if (this.editableOutline !== $event) {
       this.editableOutline = $event;
-      if (!this.chapterOutlineButtonsAreShown && $event) {
-        this.toggleChapterOutlineButtons();
-      }
       this.changeDetectorRef.detectChanges();
     }
   }
 
-  toggleChapterOutlineButtons(): void {
-    this.chapterOutlineButtonsAreShown = !this.chapterOutlineButtonsAreShown;
+  onSaveButtonClicked(): void {
+    this.outlineEditViewIsShown = this.oldOutline;
```

Comment:
Something feels wrong with this statement -- the first item is a boolean but oldOutline doesn't look like a boolean. I think some additional clarity is needed here.